pkg/command/plugin_search.go

```diff
@@ -85,7 +78,7 @@ func newSearchPluginCmd() *cobra.Command {
 	f.StringVarP(&pluginName, "name", "n", "", "limit the search to plugins with the specified name")
 	f.StringVarP(&outputFormat, "output", "o", "", "output format (yaml|json|table)")
 	f.StringVarP(&local, "local", "l", "", "path to local plugin source")
-	f.StringVarP(&targetStr, "target", "t", "", "list plugins for the specified target (kubernetes[k8s]/mission-control[tmc])")
+	f.StringVarP(&targetStr, "target", "t", "", "limit the search to plugins of the specified target (k8s/tmc/global)")
```

Comment:
Does the search work with `kubernetes` and `mission-control` target values? If not, we should support that. We can convert them to a shorter format in the CLI if needed.

Reply:
They do work with the long string like in the other commands, it's just that the flag text was getting long when I added the "global" option also. But I can add them back if you prefer.

P.S. we should add "global" in the help text of the other similar flags.

Reply:
Ok, I put it back and updated the PR description.  I looks like this:
```
  -t, --target string   limit the search to plugins of the specified target (kubernetes[k8s]/mission-control[tmc]/global)
```

I'll open a quick PR to add "global" to the other target flags.